src/seedu/addressbook/AddressBook.java

```diff
@@ -551,6 +555,24 @@ private static String executeListAllPersonsInAddressBook() {
         showToUser(toBeDisplayed);
         return getMessageForPersonsDisplayedSummary(toBeDisplayed);
     }
+    
+    /**
+     * Displays all persons in the address book to the user; in alphabetical order.
+     *
+     * @return feedback display message for the operation result
+     */
+    private static String executeSortAllPersonsInAddressBook() {
+        ArrayList<String[]> toBeDisplayed = getAllPersonsInAddressBook();
+        for (int i=0; i<toBeDisplayed.size(); i++) {
```

Comment:
I believe that Java's standard library contains sorting functions. Using them means you don't have to spend time coding and debugging your own implementation. Furthermore, the standard library often has a rather respectable runtime complexity as well. Why reinvent the wheel when someone else has perfected it for you?